pages/clubs.jsx

```diff
@@ -139,10 +139,10 @@ var Quote = React.createClass({
     return(
       <div className="row">
         <div className="col-sm-offset-2 col-sm-8 col-md-offset-2 col-md-8 col-lg-offset-2 col-lg-8">
-          <Blockquote className="primary-quote" author="Mikko K, Helsinki, Finland"
```

Comment:
Weird, so is `primary-quote` never actually used in a CSS rule? I guess it seems like it isn't, according to a [GitHub search for `primary-quote`](https://github.com/mozilla/teach.webmaker.org/search?q=primary-quote&type=Code&utf8=%E2%9C%93) at least. So I guess it's a good idea to just get rid of it...